spec/flipper/gates/percentage_of_actors_spec.rb

```diff
@@ -51,5 +49,29 @@ def context(integer, feature = feature_name, thing = nil)
         end
       end
     end
+
+    context 'for fractional percentage' do
+      let(:decimal) { 0.001}
+      let(:percentage) { decimal * 100 }
+      let(:number_of_actors) { 10_000 }
+
+      let(:actors) do
+        (1..number_of_actors).map { |n| Flipper::Actor.new(n) }
+      end
+
+      subject { described_class.new }
+
+      it 'enables feature for accurate number of actors' do
+        margin_of_error = 0.02 * number_of_actors
+        expected_open_count = 100_000 * decimal
```

Comment:
Should this be `number_of_actors * decimal`? If not where is `100_000` coming from?